src/classic/platform/argparse.rs

```diff
@@ -137,7 +144,7 @@ pub struct TArgumentParserProps {
 #[derive(Debug, Clone)]
 pub struct ArgumentParser {
```

Comment:
this whole thing should probably be replaced by `clap` or something similar eventually